src/aoc2020/day2_copy.clj

```diff
@@ -0,0 +1,127 @@
+(ns aoc2020.day2-copy
+  (:require [aoc2020.util :refer [file->seq]]
+            [clojure.string :as str]))
+
+(defn parse-policy-password-str
+  [policy-password-str]
+  (let [[policy letter pwd] (str/split policy-password-str #" ")
+        [arg1 arg2] (->> (str/split policy #"-")
+                         (mapv parse-long))
+        letter (first letter)]
+    {:policy {:arg1   arg1
+              :arg2   arg2
+              :letter letter}
+     :pwd    pwd}))
+
+(defn valid-password?
+  [policy-password]
+  (let [{:keys [policy pwd]} policy-password
+        {lo     :arg1
+         hi     :arg2
+         letter :letter} policy
+        freqs (frequencies (seq pwd))
+        {letter-freq letter :or {letter-freq 0}} freqs]
+    (<= lo letter-freq hi)))
+
+(defn count-valid-password
+  [policy-password-str-s]
+  (let [policy-password-s (->> policy-password-str-s
+                               (map parse-policy-password-str))]
+    (->> policy-password-s
+         (filter valid-password?)
+         (count))))
+
+
+(defn valid-password?
+  [policy-password]
+  (let [{:keys [policy pwd]} policy-password
+        {pos1   :arg1
+         pos2   :arg2
+         letter :letter} policy
+        [pos1-equal? pos2-equal?] (mapv (fn [pos] (= letter
+                                                     (get pwd (- pos 1))))
+                                        [pos1 pos2])
+        both?   (and pos1-equal? pos2-equal?)
+        either? (or pos1-equal? pos2-equal?)]
+    (and (not both?) either?)))
```

Comment:
Looks like my comment from last PR wasn't committed :-o

This is an unnecessarily complex interpretation of xor.